README.md

```diff
@@ -103,6 +103,9 @@ name: Publish Allure report to Morph object storage
 on:
   push:
     branches: [ main ]
+
+permissions: write-all
```

Comment:
This commit should be removed, it's meaningless.

Reply:
> This commit should be removed, it's meaningless.

Fixed